diff --git storage/mroonga/vendor/groonga/lib/grn.h storage/mroonga/vendor/groonga/lib/grn.h
index 2cd6240..38c9d35 100644
--- storage/mroonga/vendor/groonga/lib/grn.h
+++ storage/mroonga/vendor/groonga/lib/grn.h
@@ -136,19 +136,19 @@
 # ifdef __GNUC__
 #  include <stdint.h>
 # else
-typedef uint8_t UINT8;
-typedef int8_t INT8;
-typedef int_least8_t INT8;
-typedef uint_least8_t UINT8;
-typedef int16_t INT16;
-typedef uint16_t UINT16;
-typedef int32_t INT32;
-typedef uint32_t UINT32;
-typedef int64_t INT64;
-typedef uint64_t UINT64;
-typedef ssize_t SSIZE_T;
-typedef pid_t int;
-typedef off64_t int64_t;
+typedef UINT8 uint8_t;
+typedef INT8 int8_t;
+typedef INT8 int_least8_t;
+typedef UINT8 uint_least8_t;
+typedef INT16 int16_t;
+typedef UINT16 uint16_t;
+typedef INT32 int32_t;
+typedef UINT32 uint32_t;
+typedef INT64 int64_t;
+typedef UINT64 uint64_t;
+typedef SSIZE_T ssize_t;
+typedef int pid_t;
+typedef int64_t off64_t;
 # endif

 # undef MSG_WAITALL
